CSS/index.css

```diff
@@ -8,6 +8,7 @@
   position: absolute;
   left: 25px;
   cursor: pointer;
+  transition: height 2s;
```

Comment:
Without a height defined, CSS has nothing to calculate the transition with. What could you do to give this an animation?